os/arch/arm/src/amebasmart/amebasmart_spi.c

```diff
@@ -857,8 +857,11 @@ static uint16_t amebasmart_spi_send(FAR struct spi_dev_s *dev, uint16_t wd)
 	DEBUGASSERT(priv);
 
 	if (priv->role == AMEBASMART_SPI_MASTER) {
-
+#ifndef CONFIG_SPI_EXCHANGE
```

Comment:
```
static const struct spi_ops_s g_spi0ops = {
	.send         = amebasmart_spi_send,
```

amebasmart_spi_send is used by amebasmart_spi_exchange
We need a send operation that operates independently of the exchange.

Please separate the send API so that it is not related to exchange.

Reply:
I understood that need to create a new function amebasmart_send_read_disabled() ...

Reply:
@kaikaikaime Please refer other chip architecture.
imxrt
```
	.send         = imxrt_lpspi_send,
#ifdef CONFIG_SPI_EXCHANGE
	.exchange     = imxrt_lpspi_exchange,
#else
	.sndblock     = imxrt_lpspi_sndblock,
	.recvblock    = imxrt_lpspi_recvblock,
#endif
```
esp32
```
	.send = spi_send,
#ifdef CONFIG_SPI_EXCHANGE
	.exchange = spi_exchange,
#else
	.sndblock = spi_sndblock,
	.recvblock = spi_recvblock,
#endif
```

All support `send` separately from exchange.
We want the `send` regardless of the exchange.

Reply:
The `send` should not have the receive internally.

Reply:
amebasmart_spi_send will now only do internal function write and does not do return read 
internal function spi_master_exchange used in exchange will now do internal write and read 

Kai